docs/html-css/week-1/homework.md

```diff
@@ -4,37 +4,23 @@ title: Coursework
 sidebar_label: Coursework
 ---
 
-## 1) HTML Reading (1 hour)
-
-Read this page on HTML syntax. It gives you a good overview of how HTML works inside a website.
-
-https://marksheet.io/html-syntax.html
-
-## 2) Advanced CSS Selectors
-
-### 2.1) Read about CSS Selectors (1 Hour)
-
-Read this article about CSS Selectors
-
-http://learn.shayhowe.com/advanced-html-css/complex-selectors/
-
-### 2.2) CSS Diner (3 hours)
+### 1) CSS Diner (3 hours)
```

Comment:
We already cover HTML syntax and link to all CSS selectors in lesson 1, so homework 1 and 2.1 were removed. It also keeps the homework more engaging since it's now more exercise-based.